depends/patches/zeromq/cacheline_undefined.patch

```diff
@@ -0,0 +1,17 @@
+commit 8c3da001bd406673938081511f7496a8ddeeea0b
```

Comment:
Where does this commit hash come from?

I can see https://github.com/zeromq/libzmq/commit/55bd6b3df06734730d3012c17bc26681e25b549d or https://github.com/zeromq/libzmq/commit/59c91227b46b60b7f965816544b25fe0a3cc8bbe.

Reply:
From my cherry pick